### DISCUSSION TITLE
Has this project been abandoned?

### DISCUSSION THREAD

**Discussion by bigloudjeff** [AdamsLair/duality#897] [General]
Is any further work planned on it?

thanks,

Jeff

**Comment by SirePi** [comment]
Hi Jeff, unfortunately no.. Duality 3.0 is mature enough, and version 4 is a work-in-progress-that-can-be-used, but sadly no further development is foreseen for the near future.

The engine can still be used, although you have to do some manual work yourself due to the fact that an update of nuget broke compatibility with the way plugins were handled.
